scoresheets/StoringGroceries.tex

```diff
@@ -1,8 +1,11 @@
 
-The maximum time for this test is 1+3 minutes.
+The maximum time for this test is 1+2+2 minutes.
 The first minute is for the robot to open the Cupboard's door on its own.
 If the robot is not able to open the door within that minute, it will be opened by the referee. 
 In case the robot opens the door within the minute, the robot has a small time advantage. 
+Then, the robot has 2 minutes to recognize objects and manipulate items.
+If an item is manipulated within that time, 2 more minutes are added to the test time. 
```

Comment:
If robot fails to move one object within the first 3 minutes, the test ends. Nice!
However this somehow contradicts the rule for inoperative robots (30sec), so a robot should take no more than 60 seconds to place the first object in the shelf counting from the door opening. Lets say:
1+1+3 = 5